analytics/test/request.py

```diff
@@ -15,15 +15,16 @@ def test_valid_request(self):
         }])
         self.assertEqual(res.status_code, 200)
 
-    def test_invalid_write_key(self):
-        self.assertRaises(Exception,  post, 'invalid', batch=[{
+    def test_success_on_invalid_write_key(self):
```

Comment:
looking at it now, I think we can probably get rid of this test, since it's basically the same case as above